llarp/link/link_manager.cpp

```diff
@@ -534,7 +536,7 @@ namespace llarp
     catch (const std::exception& e)
     {
       log::warning(link_cat, "Exception: {}", e.what());
-      respond(serialize_response({{"STATUS", FindNameMessage::EXCEPTION}}));
+      respond(serialize_response({{messages::STATUS_KEY, FindNameMessage::EXCEPTION}}));
```

Comment:
Minor: Why do we (me included lol) still call `serialize_response` and pass two constexpr's in a dict as a parameter? Why dont we just
```C++
// messages.hpp
namespace messages {
    inline const auto STATUS_OK = serialize_response({{"!", "OK"}});
    inline const auto STATUS_EXCEPTION = serialize_response({{"!", "EXCEPTION"}});
    ....
}
```
To then invoke like:
```C++
// link_manager.cpp
respond(messages::STATUS_OK);
...
m.respond(messages::STATUS_EXCEPTION);
```
I had namespaced the messages individually when I first laid this out, but we really don't need the string `"EXCEPTION"` repeated, and the more I see `serialize_response` the more I don't want to see it lol.

Though for some cases, like `PublishIntroMessage::INVALID_ORDER`, the namespacing is useful so maybe there's a middle road where we bring as much as we can into `llarp/messages/common.hpp`.

Maybe we can even add a `bool` parameter "serialize_response" to the libquic `message::respond` method that defaults to `true`, where it does that automatically?

Reply:
Agreed, generic errors should probably be shared.  Should I chuck that in this PR?